frontend/app/controllers/modal/activity.js

```diff
@@ -40,7 +46,13 @@ export default Ember.Controller.extend({
     get() {
       let to = this.get('model.to')
 
-      return to && to.format(this.dateFormat)
+      return (to && to.format(this.dateFormat)) || moment().add(15, 'minutes').format(this.dateFormat)
```

Comment:
No need for parentheses, `&&` has higher precedence than `||` and eslint complains about unnecessary parentheses.